package.json

```diff
@@ -22,9 +22,6 @@
     "mssql": "^6.0.0",
     "strong-globalize": "^5.0.0"
   },
-  "bundledDependencies": [
-    "mssql"
-  ],
```

Comment:
Do you know why we were bundling `mssql` before? What are the ramifications of removing this bundled dependency?

From what I understand about the issue we are trying to solve, this should not be the cause of the problem, because the working version 3.6.0 was bundling `mssql` too.

Reply:
FWIW, here is the commit that introduced the bundled dependency: https://github.com/strongloop/loopback-connector-mssql/commit/9d91d9569f6da0edc8efab36460509de2f2808db

Reply:
@bajtos I'm not too familiar on the exact ramifications of having/removing mssql as a bundled dependency, but FWIW removing it seems to solve the issue.

Reply:
@bajtos I am fine with keeping the mssql bundledDependency in. @raymondfeng just mentioned to me it was to do some technical debt cleanup while we are in this area of code. But perhaps it should be part of a different github issue.

Reply:
@raymondfeng , please see @bajtos comment: https://github.com/strongloop/loopback-connector-mssql/pull/226#discussion_r388758452 .thanks.

Reply:
9d91d95 was a release commit which changed formatting of `package.json` file. The actual introduction of `bundledDependencies` was done by cad27c56ef1b1599 and is related to regenerator removal as mentioned by @raymondfeng in the PR description.